### ISSUE TITLE
Cannot use object of type PicoFeed\Config\Config as array

### ISSUE THREAD

**Issue by LunarLoony** [OleVik/grav-plugin-twigfeeds#24]
It looks like the plugin isn't loading the right config. Instead of loading from its own config file, it's trying to pull from PicoFeed's instead.

![image](https://user-images.githubusercontent.com/2440723/44151649-1ead3038-a09b-11e8-86b6-54c40e49df65.png)

I think the issue lies in the config itself, because this error doesn't happen when I use the default. It's only when I change it to the following that it breaks down (URL changed, of course).

```
enabled: true
cache: true
static_cache: false
debug: false
cache_time: 900
pass_headers: false
silence_security: false
twig_feeds:
  - source: 'https://example.com/feed'
    name: Example feed
    start: 0
    end: 6
    extra_tags:
        - "image":
            - href
        - "employer"
        - "location"
```

I'm running Grav 1.4.8, with the latest version of TwigFeed, on a Wampserver instance. SSL is not enabled on this server, in case it's relevant.

Additional information:

```
Whoops\Exception\ErrorException thrown with message "Cannot use object of type PicoFeed\Config\Config as array"

Stacktrace:
#18 Whoops\Exception\ErrorException in C:\wamp64\www\grav\user\plugins\twigfeeds\Parser.php:85
#17 TwigFeeds\Parser:parseFeed in C:\wamp64\www\grav\user\plugins\twigfeeds\twigfeeds.php:215
#16 Grav\Plugin\TwigFeedsPlugin:outputFeeds in C:\wamp64\www\grav\vendor\symfony\event-dispatcher\EventDispatcher.php:184
#15 call_user_func:{C:\wamp64\www\grav\vendor\symfony\event-dispatcher\EventDispatcher.php:184} in C:\wamp64\www\grav\vendor\symfony\event-dispatcher\EventDispatcher.php:184
#14 Symfony\Component\EventDispatcher\EventDispatcher:doDispatch in C:\wamp64\www\grav\vendor\symfony\event-dispatcher\EventDispatcher.php:46
#13 Symfony\Component\EventDispatcher\EventDispatcher:dispatch in C:\wamp64\www\grav\vendor\rockettheme\toolbox\Event\src\EventDispatcher.php:23
#12 RocketTheme\Toolbox\Event\EventDispatcher:dispatch in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:283
#11 Grav\Common\Grav:fireEvent in C:\wamp64\www\grav\system\src\Grav\Common\Twig\Twig.php:326
#10 Grav\Common\Twig\Twig:processSite in C:\wamp64\www\grav\system\src\Grav\Common\Service\OutputServiceProvider.php:27
#9 Grav\Common\Service\OutputServiceProvider:Grav\Common\Service\{closure} in C:\wamp64\www\grav\vendor\pimple\pimple\src\Pimple\Container.php:118
#8 Pimple\Container:offsetGet in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:19
#7 Grav\Common\Processors\RenderProcessor:process in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:132
#6 Grav\Common\Grav:Grav\Common\{closure} in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:374
#5 Grav\Common\Grav:Grav\Common\{closure} in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:350
#4 call_user_func_array:{C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:350} in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:350
#3 Grav\Common\Grav:__call in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:133
#2 Grav\Common\Grav:measureTime in C:\wamp64\www\grav\system\src\Grav\Common\Grav.php:133
#1 Grav\Common\Grav:process in C:\wamp64\www\grav\index.php:52
#0 {main} in C:\wamp64\www\grav\index.php:0
```

**Comment by LunarLoony** [comment]
Minor update: this also happens when there are no extra tags in the config.

**Comment by paulhibbitts** [comment]
I've also run into an issue like this @OleVik  - next time it happens I will try to take a screenshot too

**Comment by OleVik** [comment]
Sorry that this took some time to look into, please try [v3.3.3](https://github.com/OleVik/grav-plugin-twigfeeds/releases/tag/v3.3.3), it should resolve the issue.

**Comment by paulhibbitts** [comment]
Thanks @OleVik , I'll give the update a go too!

**Comment by OleVik** [comment]
Assumed fixed by v3.3.3.
